src/index.css

```diff
@@ -1,5 +1,18 @@
+*{
+  box-sizing:border-box;
+}
+
+html{
+  font-size: 62.5%;
+}
+a{
+  text-decoration: none;
+  display: inline-block;
+}
 body {
-  margin: 0;
+  height: 100%;
+  margin:20px;
```

Comment:
Try to avoid styling the body tag and instead use a div as your container.
Also, I would recommend using % or rem instead of pixels.